### PR TITLE
Update integer.py

### PR DESCRIPTION
**Pull request by MartinRakacolli**
Added Int8Field and Int16Field.

### CODE REVIEW

**Comment by moreal** [comment]
Hello @MartinRakacolli

You should write test code of int-fields you added

File: pystructs/fields/integer.py [hunk 1]
```diff
@@ -16,6 +18,13 @@ def __init__(self, size: int, byteorder='little'):
     def fetch(self) -> int:
         return int.from_bytes(super().fetch(), self.byteorder)
```
Comment (moreal) [inline pystructs/fields/integer.py]: This project follows code-style named `pep8`

Before defining functions, classes, you have to remain two blank lines

**Comment by MartinRakacolli** [comment]
Does that cover everything?

**Comment by moreal** [comment]
@MartinRakacolli yes, but you made too many useless commits. :crying_cat_face: 
could you make only two commits like 'Change to pep8 format', 'Add testcode for Int8Field, Int16Field' atomically?

you can use `git rebase -i HEAD~4` and `git commit --amend` to edit commits.   
if you need to force-push, you can do it on your repository :+1:

**Comment by MartinRakacolli** [comment]
I use Desktop Github and I can't quite figure out how to edit commits through that.

Also, while I agree it could be neater if there were only two commits, every other commit before the last one threw errors.

**Comment by moreal** [comment]
@MartinRakacolli I don't know Github Desktop as well

And we need to talk in realtime.  
I will make slack channel and add to README soon

**Comment by moreal** [comment]
@MartinRakacolli Simply, how about to use `git reset` ?

**Comment by moreal** [comment]
@MartinRakacolli I made slack invitation [link](https://pystructs-slack-application.herokuapp.com/). To talk in realtime, come here :grin:

**Comment by MartinRakacolli** [comment]
Do you still need me to work on this issue?

**Comment by moreal** [comment]
@MartinRakacolli I'll merge this pr, but in next time, I wanna study git more with you :grinning:

**Comment by moreal** [comment]
@MartinRakacolli Thanks to your contribution

**Pull request merged**